frontend/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+const starts = ['2023-04-10 13:30', '2023-05-10 12:30', '2023-06-15 14:15', '2023-06-20 14:15']
+const ends = ['2023-04-10 14:30', '2023-05-10 13:30', '2023-06-15 15:15', '2023-06-20 15:15']
+const dummy_descrip = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam dignissim efficitur sapien id facilisis. In imperdiet ante ac fermentum aliquet. Nam pellentesque vestibulum nisl quis facilisis. Sed eget magna auctor."
+const dummy_title = "This is the title of a mental health resource article";
+
+
+const Dashboard = () => {
+
+
+  return <div>
+    <div className="card w-full bg-base-100 shadow-xl">
+      <div className="card-body">
+        <h1 className="card-title pt-1/15">Upcoming Appointments</h1>
+        <div className="grid grid-cols-4 gap-4 items-center pb-1/12 shrink">
+          <AppointmentCard p_name={names[0]} start={new Date(starts[0])} end={new Date(ends[0])} description={dummy_descrip}>
+          </AppointmentCard>
+          <AppointmentCard p_name={names[1]} start={new Date(starts[1])} end={new Date(ends[1])} description={dummy_descrip}>
+          </AppointmentCard>
+          <AppointmentCard p_name={names[2]} start={new Date(starts[2])} end={new Date(ends[2])} description={dummy_descrip}>
+          </AppointmentCard>
+          <AppointmentCard p_name={names[3]} start={new Date(starts[3])} end={new Date(ends[3])} description={dummy_descrip}>
+          </AppointmentCard>
+        </div>
+      </div>
+    </div>
+
+    <div className="card w-full bg-base-100 shadow-xl">
+      <div className="card-body">
+        <h1 className="card-title pt-1/15">My Saved Psychiatrists</h1>
+        <div className="grid grid-cols-4 gap-4 items-center pb-1/12 shrink">
+          <PyschiatristCard p_name={names[0]} p_certifications="Certifications Certifications" size={1}></PyschiatristCard>
+          <PyschiatristCard p_name={names[1]} p_certifications="Certifications Certifications" size={1}></PyschiatristCard>
+          <PyschiatristCard p_name={names[2]} p_certifications="Certifications Certifications" size={1}></PyschiatristCard>
+          <PyschiatristCard p_name={names[3]} p_certifications="Certifications Certifications" size={1}></PyschiatristCard>
+        </div>
+      </div>
+    </div>
+
+    <div className="card w-full bg-base-100 shadow-xl">
+      <div className="card-body">
+        <h1 className="card-title pt-1/15">Additional Resources</h1>
+        <div className="grid grid-cols-4 gap-4 items-center pb-1/12 shrink"></div>
+        <div className="grid grid-cols-4 gap-4 margin-top: 100 items-center pb-1/12 shrink">
+          <ArticleCard title={dummy_title} body={dummy_descrip}></ArticleCard>
+          <ArticleCard title={dummy_title} body={dummy_descrip}></ArticleCard>
+          <ArticleCard title={dummy_title} body={dummy_descrip}></ArticleCard>
+          <ArticleCard title={dummy_title} body={dummy_descrip}></ArticleCard>
```

Comment:
We should ideally be loading the data from somewhere and using `.map()` to generate these components -- I know right now y'all don't have a backend but you can also load the data from a JSON file and use `map()`, which can dynamically generate components as opposed to hard coding